Change: value18 => valueB

@package vaimo/composer-patches-target3
@version <=1.2.3

@assert vaimo/composer-patches-target3,value18,valueB

--- src/example.txt.org
+++ src/example.txt
@@ -14,7 +14,7 @@
 value14
 value15
 value16
 value17
-value18
+valueB
 value19
 value20
